projects/plutus/inc/current_sense.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+// Current sense module for the LTC2484 ADC
+
+#include "ltc_adc.h"
+#include "status.h"
+
+// Current-voltage data points
+typedef struct {
+  int32_t voltage;  // Voltage in microvolts
+  int32_t current;  // Voltage in milliamps
+} CurrentSenseValue;
+
+typedef void (*CurrentSenseCallback)(int32_t current, void *context);
+
+// User-defined points for two-point calibration
+typedef struct {
+  CurrentSenseValue zero_point;
+  CurrentSenseValue max_point;
+} CurrentSenseCalibrationData;
+
+typedef struct {
+  LtcAdcStorage *adc_storage;
+  CurrentSenseCalibrationData *line;
```

Comment:
Should replace `line` with `calib_data` or similar?

Reply:
Done